src/components/Comment/Comments.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types'; // eslint-disable-line import/no-extraneous-dependencies
+import Comment from './Comment';
+
+const Comments = ({ comments, currentArticleTitle }) => (
+  <React.Fragment>
```

Comment:
Since this was as a result of not importing Fragment like so , you can do this `import React, { Fragment } from 'react'` so you can then use just `<Fragment></Fragment>` to refactor the code in line 6.

Reply:
I agree. I'll fix and push.